tests/test_cms_models.py

```diff
@@ -25,3 +27,23 @@ def test_alert_expiration(client, alert, expiration_str, should_display):
     assert (
         test_alert.description in response.content.decode("utf-8")
     ) == should_display
+
+
+@pytest.mark.parametrize(
+    "disabled,should_display",
+    [(False, True), (True, False)],
+)
+def test_tooltips_display(client, disabled, should_display, bill, metro_subject):
+    """
+    Test that tooltips display when their target is present, and when not disabled.
+    """
+    metro_subject.build(name="Kiosks")
+
+    # Add this classification to make sure the bill is discoverable by the model manager
+    test_bill = bill.build(classification=["Board Box"], subject=["Kiosks"])
+    test_content = "A test tooltip for a subject!"
+    Tooltip.objects.create(target="Subject", content=test_content, disabled=disabled)
+
+    url = reverse("lametro:bill_detail", kwargs={"slug": test_bill.slug})
+    response = client.get(url)
+    assert (test_content in response.content.decode("utf-8")) == should_display
```

Comment:
Ideally I would've loved to have this also test whether the tooltip showed up in the search view, but had a hard time getting the `test_bill` to show up in the search results response.